github/repos_environments_test.go

```diff
@@ -426,3 +426,18 @@ func TestEnvironment_Marshal(t *testing.T) {
 
 	testJSONMarshal(t, repoEnv, want)
 }
+func TestBranchPolicy_Marshal(t *testing.T) {
```

Comment:
Please add a blank line before this test for consistency.

Reply:
changes pushed